ingest/expression_files/mtx.py

```diff
@@ -214,20 +239,13 @@ def sort_mtx(file_path, mtx_file_handler: IO) -> str:
          ----------
             new_file_path (str) : Full path of newly sorted MTX file
         """
-        file_name = ntpath.split(file_path)[1]
+        file_name = ntpath.split(file_path)[1][:-3]
```

Comment:
This has a subtle bug: `file_name` retains a period.  The [robust approach you use elsewhere](https://github.com/broadinstitute/scp-ingest-pipeline/pull/162/files#diff-523e80ea0f210d08dda74bd7279f507ef450e6aa75d444aac8bee2f6bbed2895R95) seems preferable.

Bug demo:
```
wm10d-4e7:single_cell_portal_core eweitz$ python3
Python 3.7.7 (default, Mar 10 2020, 15:43:03) 
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ntpath
>>> file_path = '/path/to/file.txt'
>>> ntpath.split(file_path)[1][:-3]
'file.'
```

Applying your other, robust approach:

```suggestion
        file_name = os.path.basename(file_path)
        file_name = os.path.splitext(base)[0]
```